patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -560,7 +560,28 @@
              {
                  return false;
              }
-@@ -2011,6 +2167,11 @@
+@@ -1971,7 +2127,19 @@
+                 for (int j4 = j3; j4 < k3; ++j4)
+                 {
+                     IBlockState iblockstate1 = this.func_180495_p(blockpos$pooledmutableblockpos.func_181079_c(l3, i4, j4));
+-
++                    
++                    Boolean result = iblockstate1.func_177230_c().isAABBInsideLiquid(this, blockpos$pooledmutableblockpos, p_72953_1_);
++                    
++                    if(result != null)
++                    {
++                        if(result)
++                        {
++                            blockpos$pooledmutableblockpos.func_185344_t();
```

Comment:
We try to keep code in patches very compact, I think the preferred minimal formatting style for this patch would be like this:
```java
 ++                    Boolean result = iblockstate1.func_177230_c().isAABBInsideLiquid(this, blockpos$pooledmutableblockpos, p_72953_1_);
 ++                    if (result != null) {
 ++                        if (result) {
 ++                            blockpos$pooledmutableblockpos.func_185344_t();
 ++                            return true;
 ++                        }
 ++                        continue;
 ++                    }
```

Reply:
As long as the additions are in a same block, it is going to be safe.